addon/models/osf-model.js

```diff
@@ -20,6 +20,12 @@ export default DS.Model.extend(HasManyQuery.ModelMixin, {
         this._super(...arguments);
         this.set('_dirtyRelationships', Ember.Object.create({}));
     },
+    /**
+     * Looks up relationship on model and returns hasManyRelationship
+     * or belongsToRelationship object.
+     *
+     * @param {String} rel, relationship on the model
+     **/
```

Comment:
Thanks!

(In the future we can also document return type etc. YUIDoc doesn't parse the source code- it just renders the things we choose to document explicitly)